drivers/ovsSwitch.go

```diff
@@ -399,7 +398,7 @@ func (sw *OvsSwitch) DeletePort(epOper *OvsOperEndpointState) error {
 	}
 
 	// Delete the Veth pairs if required
-	if useVethPair {
+	if useVethPair && !skipVethPair {
```

Comment:
@shaleman Do we need to retain this "useVethPair" constant? If not, we could just get rid of it and that will make this cleaner.